### ISSUE TITLE
exception while downloading checkpoint

### ISSUE THREAD

**Issue by mattcamp** [aws-deepracer-community/deepracer-core#2]
Hi,

I've just attempted to get a local training environment set up using your instructions and images however I'm getting an error in ROS when trying to start the distributed training:

```
Received IP from SageMaker successfully: 172.18.0.2
Using custom preset file!
Waiting for checkpoint
Got exception while downloading checkpoint An error occurred (404) when calling the HeadObject operation: Not Found
Got exception while downloading checkpoint An error occurred (404) when calling the HeadObject operation: Not Found
Got exception while downloading checkpoint An error occurred (404) when calling the HeadObject operation: Not Found
Got exception while downloading checkpoint An error occurred (404) when calling the HeadObject operation: Not Found
```

Minio seems to be working ok and Sagemaker is running fine... It's also sitting there spamming the same message over and over:

```
algo-1-rrryt_1  | 64:M 23 Apr 2019 14:46:49.741 * Ready to accept connections
algo-1-rrryt_1  | Redis server started successfully!
algo-1-rrryt_1  | Uploaded IP address information to S3: 172.18.0.4
algo-1-rrryt_1  | Downloading pretrained model into ./pretrained_checkpoint from s3://bucket/rl-deepracer-pretrained
algo-1-rrryt_1  | Checkpoint not found. Waiting...
algo-1-rrryt_1  | Checkpoint not found. Waiting...
algo-1-rrryt_1  | Checkpoint not found. Waiting...
algo-1-rrryt_1  | Checkpoint not found. Waiting...
```

I can't see anything relating to checkpoints in the minio bucket at all but I'm not sure what file it's actually looking for. 

It looks like maybe it's looking for a pretrained model in the bucket but that doesn't seem to exist... do I somehow need to create it?

Full logs from Sagemaker + ROS:
[deepracer.log](https://github.com/crr0004/deepracer/files/3108082/deepracer.log)
[sagemaker.log](https://github.com/crr0004/deepracer/files/3108083/sagemaker.log)

**Comment by mattcamp** [comment]
Closing as I figured out that all I had to do was comment out the two hyperparameters inside rl_deepracer_coach_robomaker.py and now it works fine.

Great work btw!

**Comment by crr0004** [comment]
Sorry I never got around to replying! I stupidly had notifications off and have been busy with life. Yep you're right. I need to add a note about that. I'll open this again so I am prompted to add the note

**Comment by crr0004** [comment]
I've added it to the FAQ part in the readme. Closing this off again. Thanks again for raising and solving the issue!

**Comment by joezen777** [comment]
Yeah the error was not enough sleep. I had created "containers" instead of "container". And fortunately I am happy to say that everything is working after deleting all repo folders and then re-running through the steps. The only thing that I didn't test before removing and re-setting up, was if some of my changes to the hyperparameters had screwed things up. I'll also say that it took a long time before the policy training started (couple minutes) and it ran the epochs much slower than my other computer. I changed my cpus for docker from 8 to 6 after this and that made VNC run a little better but little else. But either way it's a success. Thanks again. If you're ever in NY before AWS NY Summit, hit me up and I'll show you the Slalom office in the new World Trade Center. (I'm building a custom track in there as well.)
